collaborative_practice.js

```diff
@@ -1,3 +1,4 @@
+const { lstatSync } = require('fs');
```

Comment:
Why was this required?

Reply:
yep, it slipped, I removed it when I resolved the merge conflicts